packages/gp2-components/src/templates/OnboardingCoreDetails.tsx

```diff
@@ -0,0 +1,134 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import {
+  Card,
+  Headline3,
+  Link,
+  Paragraph,
+  pixels,
+} from '@asap-hub/react-components';
+import { gp2 as gp2Routing, logout } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import { ComponentProps } from 'react';
+import { addIcon } from '../icons';
+import { mobileQuery, nonMobileQuery } from '../layout';
+import { UserDetailHeader } from '../organisms';
+import EmailSection from '../organisms/EmailSection';
+
+const { onboarding } = gp2Routing;
+type OnboardingCoreDetailProps = Pick<gp2Model.UserResponse, 'email'> &
+  ComponentProps<typeof UserDetailHeader>;
+const { rem } = pixels;
+
+const contentStyles = css({
+  paddingTop: rem(32),
+});
+
+const footerStyles = css({
+  display: 'flex',
+  justifyContent: 'space-between',
+  flexDirection: 'column-reverse',
+  [nonMobileQuery]: {
+    flexDirection: 'row',
+  },
+});
+const footerEditStyles = css({
+  display: 'flex',
+  gap: rem(32),
+  [mobileQuery]: {
+    flexDirection: 'column-reverse',
+    gap: rem(24),
+  },
+});
+const signOutStyles = css({
+  [mobileQuery]: {
+    paddingTop: rem(24),
+  },
+});
+
+const buttonStyles = css({
+  [mobileQuery]: {
+    flexDirection: 'column-reverse',
+    gap: rem(24),
+  },
+});
+
+const rowStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  [nonMobileQuery]: {
+    flexDirection: 'row',
+    gap: rem(24),
+  },
+});
+const editButtonStyles = css({
+  [nonMobileQuery]: {
+    marginLeft: 'auto',
+  },
+});
+const descriptionStyles = css({ paddingBottom: rem(32) });
+const cardStyles = css({ padding: `${rem(32)} 0` });
+const OnboardingCoreDetails: React.FC<OnboardingCoreDetailProps> = ({
+  email,
+  ...headerProps
+}) => (
+  <div css={contentStyles}>
+    <div css={descriptionStyles}>
+      In order to join the platform, we need to capture some core information
+      before you start exploring.
+    </div>
+    <UserDetailHeader {...headerProps} edit={'/edit'} />
+    <div css={cardStyles}>
```

Comment:
we are adding a lot of padding between elements, can't we just use an gap between them?